Route/RoutePageGenerator.php

```diff
@@ -99,9 +99,20 @@ public function update(SiteInterface $site, OutputInterface $output = null)
         }
 
         // Iterate over declared routes from the routing mechanism
+        $routesDedupInfos = array();
+        // We first de-duplicate routes by path; the first declaration win !
         foreach ($this->router->getRouteCollection()->all() as $name => $route) {
-            $name = trim($name);
+            $routePath = $route->getPath();
+            if (array_key_exists($routePath, $routesDedupInfos)) {
+                $this->writeln($output, sprintf('  <info>IGNORE</info> route %s is ignored because it uses the same path as %s (path=%s)',
+                    $name, $routesDedupInfos[$routePath][0], $routePath));
```

Comment:
I must agree I do not know ... ;-) I know these things are possible for Java environment.

You are just exhausting me with the administrative part of the process ; I took time to handle some problems ; the rest should be fluent and is not ; it is actually taking quiet the same time of "good time with better value" : time to code.

These things should be handled by a program, not by (wo)men.